packages/trader/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -160,16 +160,20 @@ AccountSignup.propTypes = {
     residence_list: PropTypes.array,
 };
 
-const AccountSignupModal = ({ is_visible, onSignup, residence_list, toggleAccountSignupModal, enableApp }) => {
+const AccountSignupModal = ({ is_visible, onSignup, residence_list, toggleAccountSignupModal, enableApp, disableApp, is_loading }) => {
     return (
-        <FullPageModal is_visible={is_visible}>
+        <Dialog 
+            is_visible={is_visible} 
+            disableApp={disableApp}
+            enableApp={enableApp}
+            is_loading={is_loading}>
             <AccountSignup
                 onSignup={onSignup}
                 residence_list={residence_list}
                 isModalVisible={toggleAccountSignupModal}
                 enableApp={enableApp}
             />
-        </FullPageModal>
+        </Dialog>
     );
 };
 
```

Comment:
Should we define propTypes for `toggleAccountSignupModal`, `enableApp`, `disableApp`, and `is_loading`?